environment/learnrails/spec/models/comments_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, type: :model do
```

Comment:
you are testing here model 
https://github.com/Gabz09/Working-one/blob/master/environment/learnrails/app/models/comment.rb

so filename should be environment/learnrails/spec/models/comment_spec.rb and describe should be
`RSpec.describe Comment, type: :model do `